packages/jupyterlab-lsp/src/features/highlights.ts

```diff
@@ -145,6 +145,11 @@ export class HighlightsCM extends CodeMirrorIntegration {
       return;
     }
 
+    if (root_position == null) {
+      this.console.warn('no root position available');
```

Comment:
```suggestion
      console.warn('no root position available');
```

No console wrapper on 2.x